documentation.html

```diff
@@ -1309,6 +1315,182 @@ <h3 class="component-subheading">Spaced List Group</h3>
         </div>
 
         <!-- End - List component  -->
+
+        <!-- Start - Navigation Component  -->
+
+        <div class="component" id="component-navigation">
+          <div class="container">
+            <h1 class="component-heading">Navigation</h1>
+            <p class="component-description">
+              Navigation can be use to navigate between different pages.
+            </p>
+
+            <hr class="line-styling" />
+            <!-- Simple Navigation -->
+            <h3 class="component-subheading">Simple Navigation</h3>
+
+            <div class="navigation-component-container">
+              <header>
+                <nav class="navbar box-shadow">
+                  <div class="left-navbar">
+                    <p class="logo"><a href="#">Chalk</a></p>
+                  </div>
+                  <ul class="right-navbar">
+                    <li class="NoShow"><a href="#">Home</a></li>
+                    <li class="NoShow"><a href="#">Documentation</a></li>
```

Comment:
classname "NoShow" should not be like this. please make it like other classnames that you are using like "no-show"

Reply:
Done the Changes 
Thank you For Review

Reply:
Thank You For Review